File: excelSheetColumnTitle.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string s = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    	int length = s.length();
    	string result = "";
    	while (n > 0) {
    		n --;
    		result = s[n%length] + result;
    		n /= length;
    	}
    	return result;
    }
};

File: excelSheetColumnTitle.java
public class Solution {
    public String convertToTitle(int n) {
        String s = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
    	int length = s.length();
    	String result = "";
    	while (n > 0) {
    		n --;
    		result = s.charAt(n%length) + result;
    		n /= length;
    	}
    	return result;
    }
}